GameServer/Logic/Config.cs

```diff
@@ -128,12 +128,34 @@ public class PlayerConfig
         public string Summoner2 { get { return (string)_playerData.SelectToken("summoner2"); } }
         public short Ribbon { get { return (short)_playerData.SelectToken("ribbon"); } }
         public int Icon { get { return (int)_playerData.SelectToken("icon"); } }
+        public Dictionary<string, List<int>> Runes { get { return _runeList; } }
```

Comment:
Instead of using `Dictionary<string, List<int>>`, you should create your own class. Will make the code a lot cleaner when you take a parameter of `RuneCollection` to a function instead of `Dictionary<string, List<int>>`